.bazelci/presubmit.yml

```diff
@@ -575,6 +575,14 @@ tasks:
     test_flags:
       - "--@rules_rust//rust/toolchain/channel=nightly"
       - "--@rules_rust//:no_std=alloc"
+  generated_inputs_external_repo_ubuntu2004:
```

Comment:
Instead of adding a new job, could you not instead make a repository rule that writes a small target to an external repo and add it to https://github.com/bazelbuild/rules_rust/blob/0.27.0/test/deps.bzl ?

Reply:
I think I did the right thing here https://github.com/bazelbuild/rules_rust/pull/2138/commits/8d0b467a0d2c5d6b392adcbb433871a829f17fa9. 

FYI. If the rust library is created in the root of the repository, then current implementation doesn't fail.